pkg/v1/sdk/capabilities/discovery/tkg/capabilities_test.go

```diff
@@ -47,6 +47,42 @@ func TestHasNSX(t *testing.T) {
 	}
 }
 
+func TestIsTKGS(t *testing.T) {
+
+	discoveryClientWithTKC := func() (*DiscoveryClient, error) {
+		return newFakeDiscoveryClient(tanzuRunAPIResourceList, Scheme, nil)
+	}
+
+	discoveryClientWithoutTKC := func() (*DiscoveryClient, error) {
+		return newFakeDiscoveryClient([]*metav1.APIResourceList{}, Scheme, nil)
+	}
+
+	testCases := []struct {
+		description       string
+		discoveryClientFn func() (*DiscoveryClient, error)
+		errExpected       bool
+		want              bool
+	}{
+		{"TKGS exists", discoveryClientWithTKC, false, true},
```

Comment:
I think a test case that has multiple versions of TKC API would be useful. Currently, `tanzuRunAPIResourceList` only has `v1alpha1`.

Reply:
Added v1aplha2 and v1alpha3 versions.